x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -140,6 +140,12 @@ public enum Cap {
          */
         CASE_MV,
 
+        /**
+         * Support for loading values over enrich. This is supported by all versions of ESQL but not
+         * the unit test CsvTests.
+         */
+        ENRICH_LOAD,
```

Comment:
nit: Maybe we want to turn the other features into caps as well just to keep number of divergent lines between 9.0 and 8.x minimal?

We can still remove old caps for good later.

Reply:
I *think* it's probably ok not to. It's going to be pretty rare that we make changes to the tests with these so I don't *think* the easier backports is going to come up.